docs/tempo/website/getting-started/_index.md

```diff
@@ -6,30 +6,32 @@ weight: 100
 # Getting started with Tempo
 
 Distributed tracing visualizes the lifecycle of a request as it passes through
-an application. There are a few components that must be configured in order to get a
+a set of applications. There are a few components that must be configured in order to get a
 working distributed tracing visualization.
 
-## 1. Client instrumentation
+An example trace visualized in Grafana:
 
-#### OpenTelemetry Instrumentation SDKs
+<p align="center"><img src="../example-trace.png" alt="Example Trace"></p>
+
+This document discusses the four major pieces necessary to build out a tracing system: 
+Instrumentation, Pipeline, Backend and Visualization. If one were to build a diagram laying 
+out these pieces it may look something like this:
+
+<p align="center"><img src="../getting-started.png" alt="Tracing Overview"></p>
+
+## 1. Instrumentation
+
+#### Instrumentation SDKs
 
 The first building block to a functioning distributed tracing visualization pipeline
-is client instrumentation, which is the process of adding instrumentation points in the application that collects telemetry information. 
-
-SDKs that are available in the most commonly used programming languages are listed below -
-
-* [OpenTemeletry CPP](https://github.com/open-telemetry/opentelemetry-cpp)
-* [OpenTemeletry Java](https://github.com/open-telemetry/opentelemetry-java)
-* [OpenTemeletry JS](https://github.com/open-telemetry/opentelemetry-js)
-* [OpenTemeletry .NET](https://github.com/open-telemetry/opentelemetry-dotnet)
-* [OpenTemeletry Lambda](https://github.com/open-telemetry/opentelemetry-lambda)
-* [OpenTemeletry Go](https://github.com/open-telemetry/opentelemetry-go)
-* [OpenTemeletry Python](https://github.com/open-telemetry/opentelemetry-python)
-* [OpenTemeletry Ruby](https://github.com/open-telemetry/opentelemetry-ruby)
-* [OpenTemeletry Swift](https://github.com/open-telemetry/opentelemetry-swift)
-* [OpenTemeletry Ruby](https://github.com/open-telemetry/opentelemetry-ruby)
-* [OpenTemeletry PHP](https://github.com/open-telemetry/opentelemetry-php)
-* [OpenTemeletry Rust](https://github.com/open-telemetry/opentelemetry-rust)
+is client instrumentation, which is the process of adding instrumentation points in the application that 
+creates and offloads spans. 
+
+SDKs that are available in the most commonly used programming languages are listed below.
```

Comment:
```suggestion
Below is a list of the most popular frameworks used for client instrumentation. Each of these have SDKs in most commonly used programming languages and you should carefully pick one according to your application needs. Its worth noting that OpenTelemetry is the converging standard ...
```

Since we're taking the language SDKs off, should we replace with something like this ^ ?

Reply:
Will spruce it up a bit :+1: